lib/modules/manager/npm/post-update/index.ts

```diff
@@ -226,13 +226,6 @@ export async function writeExistingFiles(
     if (yarnLock && config.reuseLockFiles === false) {
       await deleteLocalFile(yarnLock);
     }
-    // istanbul ignore next
-    if (
-      packageFile.managerData.pnpmShrinkwrap &&
-      config.reuseLockFiles === false
```

Comment:
yarn also has this logic, I don't think only pnpm should be removed

Reply:
yes, see other PR